py3status/core.py

```diff
@@ -289,9 +289,9 @@ def notify_user(self, msg, level='error'):
         We also make sure to log anything to keep trace of it.
         """
         if not self.config['dbus_notify']:
-            msg = 'py3status: {}. '.format(msg)
+            msg = 'py3status: {}.'.format(msg)
         if level != 'info':
-            msg += 'please try to fix this and reload i3wm (Mod+Shift+R)'
+            msg += ' Please try to fix this and reload i3wm (Mod+Shift+R)'
```

Comment:
OK, I was still on the nag bar so couldn't work out what you meant.  Fixed slightly differently.